src/main/scala/cromwell/engine/backend/jes/JesRuntimeInfo.scala

```diff
@@ -21,11 +21,11 @@ object JesRuntimeInfo {
     val runtimeAttributes: RuntimeAttributes = call.task.runtimeAttributes
 
     new Resources()
-      .setRamGb(runtimeAttributes.memoryGB)
-      .setCpu(runtimeAttributes.cpu)
-      .setPreemptible(runtimeAttributes.preemptible)
-      .setZones(runtimeAttributes.defaultZones.asJava)
-      .setDisks(runtimeAttributes.defaultDisks.asJava)
+      .setRamGb(runtimeAttributes.memoryGB.gb)
+      .setCpu(runtimeAttributes.cpu.count)
+      .setPreemptible(runtimeAttributes.preemptible.isPreemptible)
+      .setZones(runtimeAttributes.defaultZones.zones.asJava)
+      .setDisks(runtimeAttributes.defaultDisks.disks.asJava)
```

Comment:
There's a definite `thing.thingy` pattern here that would be nice to eliminate if possible